store_test.go

```diff
@@ -485,3 +486,107 @@ func TestWithSplitStore(t *testing.T) {
 	store.Free()
 	store.Free()
 }
+
+func TestStoreList(t *testing.T) {
+	reexec.Init()
+	wd := t.TempDir()
+
+	pullOpts := map[string]string{"Test1": "test1", "Test2": "test2"}
+	store, err := GetStore(StoreOptions{
```

Comment:
To the extent this is non-trivial, it should be a shared utility, not copied in each test.

Maybe a `newTestStore(*testing.T, StoreOptions)` where `StoreOptions` contains only the non-obvious specific items (like `PullOptions` or `ImageStore` in this file), and the `newTestStore` creates options with the other relevant values.